### ISSUE TITLE
New nbconvert release requested

### ISSUE THREAD

**Issue by JasonWeill** [jupyter/foundations-and-standards-team-compass#6]
Back on October 16, @janpfeifer requested a new release of nbconvert: https://github.com/jupyter/nbconvert/issues/2185

It looks like the most recent version is v7.16.4, which was cut on April 29, and there have been six commits since then. Can one of the team's members please cut a new nbconvert release? Thanks!

**Comment by krassowski** [comment]
@Carreau could you possibly help with this?

**Comment by Carreau** [comment]
I bumped you to admin; but likely the workflow should require maintain, not admin.

And there need to be a better plan to maintain nbconvert.

**Comment by krassowski** [comment]
> but likely the workflow should require maintain, not admin

Agreed, admin should not be needed for release. We can make dedicated "Release" teams and restrict the release right to them.

**Comment by Carreau** [comment]
> > but likely the workflow should require maintain, not admin
> 
> Agreed, admin should not be needed for release. We can make dedicated "Release" teams and restrict the release right to them.

I think the maintain team is enough, it probably should be the action that checks for maintains and not admin.
